src/material/sidenav/sidenav.ts

```diff
@@ -114,6 +114,10 @@ export class MatSidenav extends MatDrawer {
   encapsulation: ViewEncapsulation.None,
 })
 export class MatSidenavContainer extends MatDrawerContainer {
-  @ContentChildren(MatSidenav) _drawers: QueryList<MatSidenav>;
+  @ContentChildren(MatSidenav, {
+    // We need to use `descendants: true`, because Ivy will no longer match
+    // indirect descendants if it's left as false.
+    descendants: true
```

Comment:
@mmalerba this can technically break if people nest sidenav containers into each other and would need some extra logic to work correctly. Is nesting them something we support?

Reply:
I don't think anything in our documentation talks about it, but it does work. And I'm fairly certain someone out there is using it for a stacked drawer effect like this: https://stackblitz.com/edit/angular-jkvezt?file=app/sidenav-drawer-overview-example.html